tibco/src/test/java/org/frankframework/extensions/tibco/pipes/ObfuscatePipeTest.java

```diff
@@ -22,18 +18,18 @@
 
 class ObfuscatePipeTest extends PipeTestBase<ObfuscatePipe> {
 
-	private final String plainText = "Bacon ipsum dolor amet chuck pork.";
+	private static final String PLAIN_TEXT = "Bacon ipsum dolor amet chuck pork.";
 
 	@Override
 	public ObfuscatePipe createPipe() {
 		return new ObfuscatePipe();
 	}
 
 	@Test
-	void testEncryption() throws ConfigurationException, PipeStartException, IOException, PipeRunException {
+	void testEncryption() throws Exception {
 		// Arrange
 		configureAndStartPipe();
-		byte[] inputString = plainText.getBytes(StandardCharsets.UTF_8);
+		byte[] inputString = PLAIN_TEXT.getBytes(StandardCharsets.UTF_8);
 		Message in = new Message(inputString);
```

Comment:
Kan je niet direct de message in :D

Reply:
direct de string gebruiken bedoel je denk ik? Aangepast